docs/usage/validation_errors.md

```diff
@@ -1369,7 +1371,7 @@ except ValidationError as exc_info:
 
 ## `string_type`
 
-This error is raised when the value type is not string.
+This error is raised when the value type is not a string.
```

Comment:
```suggestion
This error is raised when the value is not a string.
```
(You can say the type is string/`str`, or the value is _a_ string. But I wouldn't say the "value type is a string")